addons/io_scene_gltf2/io/exp/gltf2_io_user_extensions.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 The glTF-Blender-IO authors.
```

Comment:
Is this copyright correct or should I put another thing?

Reply:
All files should have the same header, I think – so just 2018-2019 here.